examples/gno.land/r/sys/users/users.gno

```diff
@@ -0,0 +1,73 @@
+package users
+
+import (
+	"std"
+
+	"gno.land/p/demo/avlhelpers"
+	susers "gno.land/p/sys/users"
+)
+
+func ResolveName(username string) *susers.User {
+	if username == "" {
+		return nil
+	}
+
+	raw, ok := uStore.Get(username)
+	if !ok {
+		return nil
+	}
+
+	return raw.(*susers.User)
+}
+
+func ResolveAddress(addr std.Address) *susers.User {
+	if !addr.IsValid() {
+		return nil
+	}
+
+	raw, ok := aStore.Get(addr.String())
+	if !ok {
+		return nil
+	}
+
+	return raw.(*susers.User)
+}
+
+// ResolveAddressOrName
+// Note: unlike ResolveName, input must be "@" prefixed for names.
+func ResolveAddressOrName(aon susers.AddressOrName) *susers.User {
+	name, isName := aon.Name()
+	if isName {
+		return ResolveName(name)
+	}
+
+	return ResolveAddress(std.Address(aon))
+}
+
+// Users returns a paginated list of users
+func Users(offset, count uint64) []susers.User {
```

Comment:
let's use the upcoming `avlreader` to either expose it directly or update this method to use `avl.Pager`.

Reply:
Exposed the ROAVL